src/components/inputs/date-input/date-input.js

```diff
@@ -215,7 +216,11 @@ const DateInput = props => {
                         key={item}
                         isToday={isSameDay(today, item)}
                         {...getItemProps({
-                          disabled: props.isDisabled,
+                          disabled: !isDateInRange(
```

Comment:
Will this mean that `props.isDisabled` will not affect the disable state?

Reply:
It was wrong before. When the component `isDisabled`, the calendar menu is never opened/rendered, so there was no point disabling anything inside of it.